Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -111,6 +117,30 @@ async function run() {
                 await swapSlot(endPoint, resourceGroupName, webAppName, sourceSlot, swapWithProduction, targetSlot, preserveVnet);
                 break;
             }
+            case "Enable Continuous Monitoring": {
+                var appInsightsManage = new appInsightsManageUtils.AppInsightsManage(endPoint, appInsightsResourceGroupName, appInsightsResourceName, webAppName, resourceGroupName, specifySlotFlag, slotName);
+                await appInsightsManage.configureAppInsights();
+                break;   
+            }
+            case "Start Webjobs": {
+                resourceGroupName = (specifySlotFlag ? resourceGroupName : await azureRmUtil.getResourceGroupName(endPoint, webAppName));
+                var publishingProfile = await azureRmUtil.getAzureRMWebAppPublishProfile(endPoint, webAppName, resourceGroupName, specifySlotFlag, slotName);
+                var continuousJobs = await azureRmUtil.getAllContinuousWebJobs(publishingProfile);
+                for(var continuousJob of continuousJobs) {
+                    await azureRmUtil.startContinuousWebJob(publishingProfile, continuousJob.name);
+                }
+                break;
+            }
+            case "Stop Webjobs": {
+                resourceGroupName = (specifySlotFlag ? resourceGroupName : await azureRmUtil.getResourceGroupName(endPoint, webAppName));
+                var publishingProfile = await azureRmUtil.getAzureRMWebAppPublishProfile(endPoint, webAppName, resourceGroupName, specifySlotFlag, slotName);
+                tl.debug('Retrieved publishing Profile');
+                var continuousJobs = await azureRmUtil.getAllContinuousWebJobs(publishingProfile);
```

Comment:
Should we stop triggered jobs too?
If not, some may be triggered in-between deployments?

Reply:
Trigger jobs are targeted to run a single time and then die over. And there is not REST API too to stop triggered jobs.